### ISSUE TITLE
Rules around "fully supported" seem inconsistent

### ISSUE THREAD

**Issue by jensimmons** [Fyrd/caniuse#6923]
I was just looking up support information about Web GPU, and noticed that Chrome has a footnote explaining that it's fully supported, but not on Linux.

<img width="1254" alt="Screenshot 2023-12-18 at 2 17 53 PM" src="https://github.com/Fyrd/caniuse/assets/108474/5ac4b3fa-2d2a-459d-84e3-440cea166d18">

This surprised me. I thought Can I Use refuses to give a browser credit for full support until it's available on every operating system. 

I've certainly made the case in the past that it does make sense to me to say "this technology is fully implemented correctly, so is bright green, _but_ do make sure you have the right OS". But that line of thinking has been overruled multiple times when I've advocated for fully supported on the charts for other technology.

Can you explain the inconsistency?

**Comment by Fyrd** [comment]
Yeah, mostly because desktop Linux has such a low user share and I don't think I've heard of anyone specifying that they feel support should be partial because of this. That said it seems reasonable enough to consider it partial support so I'm okay with updating. 

Also as previously mentioned I've been more lax in the past with considering features as being "supported", so some of these are probably worth revisiting. There don't seem to currently be any other features missing support only on Linux though.
